generator/internal/service_code_generator.h

```diff
@@ -144,10 +143,9 @@ class ServiceCodeGenerator : public GeneratorInterface {
   VarsDictionary service_vars_;
   std::map<std::string, VarsDictionary> service_method_vars_;
   std::vector<std::string> namespaces_;
-  std::vector<std::reference_wrapper<google::protobuf::MethodDescriptor const>>
-      methods_;
-  std::vector<std::reference_wrapper<google::protobuf::MethodDescriptor const>>
-      async_methods_;
+  bool emit_backwards_compatibility_namespace_alias_ = false;
```

Comment:
naming nit: I don't like `emit`. I think it is too similar to `omit`, which has the opposite meaning. I prefer `declare`. Other names could be: `define` / `include` / `append`

Your call.

Reply:
Changed to `define_`.